boards/arm/mm_feather/mm_feather.yaml

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2021, MADMACHINE LIMITED
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: mm_feather
+name: MM MM-FEATHER
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 32768
+flash: 8192
+supported:
+  - counter
+  - sdhc
```

Comment:
There are a few features you've enabled on the board that are missing from this list. See `boards/arm/mimxrt1060_evk/mimxrt1060_evk.yaml` for example. This will enable twister to select more sample and test applications for this board in CI.

Reply:
@MaureenHelm  Thanks for review. Twister seems to need actual hardware and matching test code under tests/driver to perform CI test. If just add these features to the board metadata file without adding test code, will this cause CI test problems?

Reply:
Add missing features for mm_feather.yaml, Please review, Thanks